src/seth/libexec/seth/seth-source

```diff
@@ -19,7 +19,11 @@ set -e
 
 ADDR=$(seth --to-address "$1")
 
-RESPONSE=$(seth curl "http://api-cn.etherscan.com/api?module=contract&action=getsourcecode&address=${ADDR}&apikey=${ETHERSCAN_API_KEY}" | seth --show-json)
+URL="https://api.etherscan.io"
```

Comment:
I think you could do
```suggestion
URL="${ETHERSCAN_API_URL:-https://api.etherscan.io}"
```

Reply:
https://www.tldp.org/LDP/abs/html/parameter-substitution.html

Reply:
nice thx